src/Monitor/Monitor/help/Remove-AzDataCollectionRule.md

```diff
@@ -49,13 +49,13 @@ Remove-AzDataCollectionRule
 ## DESCRIPTION
 The **Remove-AzDataCollectionRule** cmdlet delete a data collection rule.
 
-Data Collection Rules (DCR) define data coming into Azure Monitor and specify where that data should be sent or stored. Here is the complete [DCR overview article](https://docs.microsoft.com/azure/azure-monitor/platform/data-collection-rule-overview).
+Data Collection Rules (DCR) define data coming into Azure Monitor and specify where that data should be sent or stored. Here is the complete [DCR overview article](/azure/azure-monitor/essentials/data-collection-rule-overview).
```

Comment:
Hi @mikefrobbins if changing to relative path, user wouldn't be able to use the link via in-tool help AKA `get-help`. Do you think we can keep the hostname?

Reply:
I've changed the URL's back to using a FQDN as they do show with `Get-Help`. This will impact air gapped environments, but I believe it's the best compromise.